_includes/forms/required-label.html

```diff
@@ -0,0 +1 @@
+<span class="required-label">*</span>
```

Comment:
```suggestion
<span class="required-label" aria-hidden="true">*</span>
```

Not all screen readers will correctly handle this, but for those that do, hide the asterisk. Let `aria-required` dictate that the field is required